src/__tests__/__snapshots__/fjs_babel-test.js.snap

```diff
@@ -457,6 +459,20 @@ exports[`test if-line-comment.js 1`] = `
 "
 `;
 
+exports[`test jsx.js 1`] = `
+"var single = \'quotes\';
+var outnumber = \'double\';
+var moreSingleQuotesThanDouble = \'!\';
+
+React.createClass({ render() {
```

Comment:
We should probably break the full block if the inner content has some kinda break in it..?